harvester/oncokb.py

```diff
@@ -24,27 +25,37 @@ def harvest(genes):
             genes.append(gene['hugoSymbol'])
     for gene in set(genes):
         gene_data = {'gene': gene, 'oncokb': {}}
-        url = 'http://oncokb.org/api/private/search/variants/clinical?hugoSymbol={}'  # NOQA
-        url = url.format(gene)
-        r = requests.get(url)
-        if r.status_code != 200:
-            print "{} {} {}".format(gene, url, r.status_code)
+        clin_url = 'http://oncokb.org/api/private/search/variants/clinical?hugoSymbol={}'  # NOQA
+        clin_url = clin_url.format(gene)
+        clin_r = requests.get(clin_url)
+        if clin_r.status_code != 200:
+            print "{} {} {}".format(gene, clin_url, clin_r.status_code)
         else:
-            gene_data['oncokb']['clinical'] = r.json()
+            gene_data['oncokb']['clinical'] = clin_r.json()
             for clinical in gene_data['oncokb']['clinical']:
                 key = "LEVEL_{}".format(clinical['level'])
                 if key in levels:
                     clinical['level_label'] = levels[key]
                 else:
                     print '{} not found'.format(clinical['level'])
```

Comment:
This is the difference between the "clinical" and "biological" entries in OncoKB. Those lacking a level label would fall under "biological".